app/services/twitter_service.rb

```diff
@@ -0,0 +1,16 @@
+class TwitterService
+  attr_reader :client
+
+  def initialize
+    @client ||= Twitter::REST::Client.new do |config|
+      config.consumer_key        = ENV['TWITTER_CONSUMER_KEY']
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.